src/renderer/views/Channel/Channel.js

```diff
@@ -90,18 +91,21 @@ export default Vue.extend({
       return this.$store.getters.getActiveProfile
     },
 
-    isSubscribed: function () {
+    subscriptionInfo: function () {
       const subIndex = this.activeProfile.subscriptions.findIndex((channel) => {
         return channel.id === this.id
       })
-
-      if (subIndex === -1) {
-        return false
+      if (subIndex !== -1) {
+        return this.activeProfile.subscriptions[subIndex]
```

Comment:
Maybe just 
```js
return this.activeProfile.subscriptions.find((channel) => {
  return channel.id === this.id
})
```
But this returns `undefined` if not found

Reply:
You can make it return `null` instead of `undefined` by using the "nullish" operator:

```js
return this.activeProfile.subscriptions.find((channel) => {
  return channel.id === this.id
}) ?? null
```